stable/kong/Chart.yaml

```diff
@@ -10,5 +10,5 @@ maintainers:
 name: kong
 sources:
 - https://github.com/Kong/kong
-version: 0.11.2
+version: 0.12.2
```

Comment:
Please bump to 0.12.0

Reply:
Fixed in https://github.com/helm/charts/pull/14509/commits/041d75feb0cd119d4ba7a0bb2dfe689288e1c381